src/main/java/com/google/sps/servlets/UpdateInfoServlet.java

```diff
@@ -59,22 +59,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String bgColor;
     String textColor;
     String email;
-    name = font = fontSize = bgColor = textColor = email = null;
+    name = font = fontSize = bgColor = textColor = email = "";
 
     try {
+      // Get all values and remove all html tags and trim the spaces in the parameters.
       name = request.getParameter("name");
+      name = name.replaceAll("\\<.*?\\>", "");
```

Comment:
you could probably factor out replaceAll/trim calls into a function (e.g. processExternalText()). That way the code here would be shorter:
name = processExternalText(request.getParamenter("name"));

...and if you decide to add another transformation, you will only need to edit processExternalText() function.

Reply:
Will do!